pub-matic-htb.js

```diff
@@ -180,11 +222,11 @@ function PubMaticHtb(configs) {
         ext.wrapper.wiid = __globalConfigs.wiid || undefined; //
         //ext.wrapper.wv = Constants.REPO_AND_VERSION;
         ext.wrapper.transactionId = __globalConfigs.transactionId;
-        ext.wrapper.wp = 'pbjs' ;
+        ext.wrapper.wp = 'pbjs';
```

Comment:
why are we setting the value of ext.wrapper.wp param to 'pbjs' when we know that this code is for IndexExchnage wrapper?